### DISCUSSION TITLE
This discussion topic should not be used for other discussion topics - Discussion forum for MyTourbook

### DISCUSSION THREAD

**Discussion by wolfgang-ch** [mytourbook/mytourbook#920] [General]
I would like to froze down the discussion forum on SourceForge https://sourceforge.net/p/mytourbook/discussion/622811/ and force the user to continue on GitHub

I was unable to migrate the existing discussions to GibHub and nobody helped me https://sourceforge.net/p/mytourbook/discussion/622811/thread/e95fac98e8/ so there will be no smooth transition.

I just changed the discussion link on the MT homepage into this forum and will make the SF discussion readonly in a few days.

Any objections to this?

**Comment by FJBDev** [comment]
> Any objections to this?

This is fine by me.
I actually think it's better to have everything in 1 place rather than jumping from GitHub to SourceForge and back to GitHub etc...

**Comment by telemaxx** [comment]
one place for everything is a lot better. 

discussions and issues, both on github.

**Reply by wolfgang-ch** [reply]
> one place for everything is a lot better

Your words seems to be interpreted wrongly by creating the following replies

**Comment by taunusrenner** [comment]


**Reply by wolfgang-ch** [reply]
Can you create a separate discussion topic for this issue?

**Reply by taunusrenner** [reply]
done

**Comment by hkrudolph** [comment]
@wolfgang-ch 
I just saw your fix for the graphic problem in the Mac version (https://sourceforge.net/p/mytourbook/discussion/622811/thread/95b05d7408/#19ce/0d01) ... I installed it, and tested with a Sigma (.fit) file that showed me the problem in the old version ... it is indeed fixed. Produces overlapping curves in a single graph or separate graphs stacked on top of each other, depending of the option chosen ... Many thanks for this fix!!!

**Reply by wolfgang-ch** [reply]
It is good to hear, that this could fix it, I was not sure if this workaround can do it. 

This fix will be available in the next version but you can use this fixed version as it is identical with 22.8 + this fix

@hkrudolph 
>Produces overlapping curves in a single graph or separate graphs stacked on top of each other

can you add a screenshot here

**Reply by hkrudolph** [reply]
Here they come ...
<img width="743" alt="overlapping" src="https://user-images.githubusercontent.com/16167839/188270490-9e6d4292-a978-452b-a69a-2e391fcf9a89.png">
<img width="743" alt="stacked" src="https://user-images.githubusercontent.com/16167839/188270492-2eeb6372-6df7-4b62-b728-ed09c17d7320.png">

**Comment by FJBDev** [comment]
@wolfgang-ch 
You should also update the link on the MT's website that points to the sourceforge forum

![image](https://user-images.githubusercontent.com/25235665/188273234-ceb5679c-2050-4ad6-90f1-861f3b42190f.png)

**Reply by wolfgang-ch** [reply]
I didn't thought about this location, it's now fixed

**Comment by FJBDev** [comment]
@wolfgang-ch Did you actually freeze down the SourceForge forum ? I just saw several new created threads on it today

**Reply by wolfgang-ch** [reply]
SF forum is now frozen, except for me :-)

**Comment by taunusrenner** [comment]
I put comment/question only on github

Am 3. September 2022 18:14:21 schrieb FJBDev ***@***.***>:

**Comment by Marty56** [comment]
Hi Wolfgang,

Thanks for providing this fix. Tour Grafik is working, Hurrah!!

Unfortunately the tour track is not displayed in the map anymore.
And also the openstreetview maps are missing. 
The only map working (working means without displaying the track) is mtbmap.cz which is pretty bad.

**Comment by Marty56** [comment]
I fiddled around with some settings,  not knowing what I am doing, and it seems to be working again.
It is too bad that I am too dumb to understand the convoluted user interface after all these years.
